jetty9-compat-base/src/main/jetty-base/modules/gae.mod

```diff
@@ -39,6 +39,7 @@ jetty.httpConfig.requestHeaderSize=8192
 jetty.httpConfig.responseHeaderSize=8192
 jetty.httpConfig.sendServerVersion=true
 jetty.httpConfig.sendDateHeader=false
+jetty.server.stopTimeout=30000
```

Comment:
Not commented here, but commented on the one above... jetty9-base/src/main/jetty-base/modules/gae.mod
Any reasons?